server/templates/dev/dev.html

```diff
@@ -19,7 +19,8 @@
 {% set title = 'Dev page' %}
 
 {% block head %}
-  <link rel="stylesheet" href={{url_for('static', filename='css/dev.min.css')}}>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
I think this should come after the main stylesheet

Reply:
what is the difference?

Reply:
I'm not sure actually, I was told to do this in a past pr and I assumed it was best practice.

Reply:
stylesheets are blocking resources, so the most important stylesheet should load first (the page can't really render without dev.min.css, but the icons are additive)

https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css

Reply:
Good to know. Updated all occurrence.